tests/python/utils.py

```diff
@@ -417,6 +417,35 @@ def inner_fn(*args, **kwargs):
     return inner_fn
 
 
+UPDATED_SDPA = torch.__version__ > "2.6.0"
```

Comment:
This will be updated to str(torch.__version__) > "2.7.0".

The changes to SDPA were made between versions 2.7.0 and 2.8.0 based on `version.txt` in PyTorch. So I am using `2.7.0`.